.github/workflows/presubmit.yml

```diff
@@ -36,6 +44,8 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v3
+      with:
+        fetch-depth: 0
```

Comment:
nit: is this necessary?

Reply:
No, this was a remnant of an intermediate step where I added conform to the lint step. I'll remove this. Good catch!